packages/vx-brush/src/BaseBrush.tsx

```diff
@@ -88,6 +107,15 @@ export default class BaseBrush extends React.Component<BaseBrushProps, BaseBrush
     disableDraggingSelection: false,
     clickSensitivity: 200,
     resetOnEnd: false,
+    scrubberInitialState: null,
+  };
+
+  componentDidMount = () => {
```

Comment:
can we remove the arrow functions here? (lifecycle events should be normal functions)